app/src/ui/window/window-controls.tsx

```diff
@@ -52,9 +52,9 @@ export class WindowControls extends React.Component<{}, IWindowControlState> {
 
   private onWindowStateChanged = (
     event: Electron.IpcMessageEvent,
-    args: any
+    windowState: WindowState
   ) => {
-    this.setState({ windowState: args as WindowState })
```

Comment:
The change above can also be justified by this usage of the IPC result - I just tidied it up here to avoid the `any` usage.